src/app/users/user-edit/user-edit.component.ts

```diff
@@ -61,8 +99,22 @@ export class UserEditComponent implements OnInit {
         changes: updatedOrNewUser,
       };
       this.store.dispatch(updateUser.success({ user: update }));
+      this.updatePending$ = this.store.select(updateUserPending);
     } else {
       this.store.dispatch(addUser.begin({ user: updatedOrNewUser }));
+      this.addUserPending$ = this.store.select(addUserPending);
+    }
+  }
+  getErrorMessage() {
+    return 'Enter a valid email, 45 Characters max!';
```

Comment:
Make this function dynamic.
It Should return the error string from the validator.
You should pass the control name from the Template.
on the function get the control. like.
```const ctl = this.myformGroup.get(ctlName);```
then get the errors array. 
return maybe... the first of the list.  or a custom string with all the errors formatted some way.. but, the first option is more than acceptable.

Reply:
I have problems fixing this issue. I understand the statement but cannot solve it.
Can yuo please give me  a hand?

Reply:
Sure, let's have a coding meeting later today?